README.md

```diff
@@ -234,6 +300,52 @@ At this point, the `Podman` docs claim that one should have moved over to `Podma
 1. In `$HOME/.docker/config.json` replace "credsStore" with "credStore" (minus an 's') to solve registry credentialing problems.
 2. Set `export DOCKER_HOST=<your_podman_machine_socket_path>` to the `Podman machine`'s socket on your system, which you can find in the `Resources` tab of `Podman Desktop`. The path starts with `unix://`.
 
+
+#### Option 2: Using Docker
+
+If you choose to use Docker, you'll need:
+1. Docker installed on your system: [Install Docker](https://docs.docker.com/engine/install)
+2. Docker Compose installed: [Docker Compose](https://docs.docker.com/compose/install)
+
+If the following commands return sensible results then Docker and Docker Compose are installed:
+Verify your installation with:
+```bash
+docker && docker compose || docker-compose
+```
+
+**Optional, but recommended to Install Docker Desktop**
+
+All of the build/tear-down is done with `docker compose`.
+
+To start the application, run at the top-level of the 8Knot directory:
+
+```bash
+docker compose up --build
+```
+
+Due to a known deadlock, we recommend scaling-up the number of worker-pool pods deployed.
+There need to be (#visualizations + 1) celery threads available for the callback_worker pool.
+
+A concrete example: I have 6 CPU's allocated to my Docker runtime, so Celery workers will default to a concurrency of 6 processes.
+However, there are 7 visualizations on the Overview page. Therefore, I will scale the 'callback_worker' pod to 2 instances,
+guaranteeing that there are (2 \* #CPUs = 12) available processing celery threads, ensuring that the known deadlock will be avoided.
+
+```bash
+docker compose up --build --scale worker-query=2 --scale worker-callback=2
+```
+
+To stop the application:
+
+```bash
+ctrl-c
+```
+
+To clean up the stopped containers, run:
+
+```bash
+docker compose down --volumes
+```
+
 ### Build and Run
 
 8Knot is a multi-container application.
```

Comment:
No need to add both commands. Just take the original text and change docker to podman